lib/modules/dosomething/dosomething_gladiator/dosomething_gladiator.cron.inc

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * Implements hook_cron()
+ */
+ function dosomething_gladiator_cron() {
+   $client = _dosomething_gladiator_build_http_client();
+
+   $options = [
+     'method' => 'GET',
+     'headers' => $client['headers'],
+   ];
+
+   $response = drupal_http_request($client['base_url'] . '/contests', $options);
+   $contests = json_decode($response->data)->data;
+
+   foreach ($contests as $contest) {
+     $nid = $contest->campaign->id;
+     // For some reason using the boolean value doesnt work in the fields array, have to convert to int.
+     $signupsOpen = $contest->waiting_room->open ? 1 : 0;
+
+     db_update('dosomething_signup_data_form')
+       ->fields([
+         'required' => $signupsOpen,
+         'competition_signup' => $signupsOpen,
+       ])
+       ->condition('nid', $nid)
```

Comment:
ah I see, but shouldnt that resolve itself once someone updates the campaign & cron runs again?